ti-linux-firmware/suite/bookworm/debian/firmware-cnm-wave.links

```diff
@@ -0,0 +1 @@
+/lib/firmware/cnm/wave521c_k3_codec_fw.bin /lib/firmware/cnm/wave521c_codec_fw.bin
```

Comment:
What name does the driver want, why symlink instead of just a rename.

Reply:
hm.. no reason. Just wanted to keep the original binary names intact.